docs/api/api_blueprint.apib

```diff
@@ -25641,9 +25641,9 @@ This Group's APIs can be used for adding a particular event to the favourite lis
             }
         }
 
-## Delete Favourite Events [/v1/user-favourite-events/{event_id}]
+## Delete Favourite Events [/v1/user-favourite-events/{event_identifier}]
 + Parameters
-    + event_id: 1 (integer) - ID of the Event
+    + event_identifier: 1 (string) - identifier or event id of the event. (b8324ae2 is an example of identifier)
```

Comment:
Completely wrong. Revert

Reply:
@iamareebjamal Only event_id (without identifier) is not being used anywhere in the API. Can you please help me a bit more regarding this.

Reply:
It's being used literally everywhere. Event identifier is only being used in 1 place on frontend

Reply:
@iamareebjamal In API docs, I cannot find a single example where only `event_id` is used. Can you please point me to one.